spec/spec_helper.rb

```diff
@@ -35,6 +35,13 @@
     "responders"
 
   config.filter_run :focus
+
+  if ENV["RUN_DRIVING_TEST"] == "true"
```

Comment:
_not blocking_ - Ahh this runs *only* the driving test. Maybe a rename could work to make it more about the intent (pre-deploy testing) than the implementation?

Reply:
gooood call - updated name at 5388bce